src/pages/blog/video-banner/_DBanner.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import VideoSrc from '../../markets/static/video/globe.mp4'
+import VideoCarousel from './_VideoCarousel'
+import VideoPlayer from './_video-player'
+import { Flex, Container } from 'components/containers'
+import { Header, LinkText } from 'components/elements'
+import device from 'themes/device'
+import PlayIcon from 'images/svg/blog/video/Triangle.svg'
+
+const ParentWrapper = styled(Flex)`
+    /* prettier-ignore */
+    background: ${(props) =>
+        props.bg_image
+            ? 'linear-gradient(251.14deg,rgba(14, 14, 14, 0.5632) 29.18%,rgba(7, 6, 6, 0.88) 85.14%),url(' +
+              props.bg_image +
+              ') no-repeat top left'
+            : 'linear-gradient(251.14deg, rgba(14, 14, 14, 0.5632) 29.18%, rgba(7, 6, 6, 0.88) 85.14%)'};
+    flex-direction: column;
+    overflow: hidden;
+    margin: 80px 0;
+    position: relative;
+    background-size: cover;
+    height: auto;
+    padding: 160px 0 80px;
+    max-width: 1600px;
+
+    @media ${device.tabletL} {
+        /* prettier-ignore */
+        background: ${(props) =>
+            props.bg_image
+                ? 'linear-gradient(251.14deg, rgba(14, 14, 14, 0.5632) 29.18%, rgba(7, 6, 6, 0.88) 85.14%),url(' +
+                  props.bg_image +
+                  ') no-repeat top right 46.5%'
+                : 'linear-gradient(251.14deg, rgba(14, 14, 14, 0.5632) 29.18%, rgba(7, 6, 6, 0.88) 85.14%)'};
+        background-size: cover;
+        padding: 73px 0 40px;
+    }
+`
+
+const BgWrapper = styled(Container)`
+    flex-direction: column;
+    justify-content: flex-start;
+`
```

Comment:
`Container` can take those as props by itself.

Reply:
fixed in latest commit